added more documentation of dependencies (#1002)
yeatmanlab/pyAFQ: docs/source/howto/usage/kwargs.rst

==========================================================
DATA
==========================================================
.. note::

	The b value is a factor that reflects the strength and 
	timing of the gradients used to generate diffusion-weighted images. 
	The higher the b-value, the stronger the diffusion effects. 
	The b-value depends on the strength, duration, and spacing 
	of these pulsed gradients. A larger b-value is achieved with 
	increasing the gradient amplitude and duration and by widening the 

> 	interval between gradient pulses. b-values should be expressed as 

Reviewer: ```suggestion
	interval between gradient pulses. b-values should be expressed in units of 
```

	doing dti. Only applies to dti.
	Default: False

.. note::

	Constrained Spherical Deconvolution (CSD) aims to solve an equation 
	that partials out the effect of the scanner's response function 
	and noise to estimate the underlying fiber orientation. Alternative

> 	model-free and model-based methods are not implemented within pyAFQ.  

Reviewer: Maybe instead? 

```suggestion
	a plethora of model-free and model-based methods is implemented in `DIPY <https://dipy.org>`_.
```
